app/views/layouts/_tl_options.html.haml

```diff
@@ -50,25 +53,25 @@
                             :name         => 'showDetailedEvents',
                             'ng-model'    => 'reportModel.showDetailedEvents'}
                     = _("Show Detailed Events")
-                %span{'ng-if' => 'reportModel.tl_show !== "timeline"', 'class' => 'timeline-option'}
+                %span{'ng-if' => 'vm.reportModel.tl_show !== "timeline"', 'class' => 'timeline-option'}
                   %label.radio
-                    %input{"ng-model" => "reportModel.tl_result", :type => "radio", :value => "success"}
+                    %input{"ng-model" => "vm.reportModel.tl_result", :type => "radio", :value => "success"}
                     = _("Successful Events")
                   %label.radio
-                    %input{"ng-model" => "reportModel.tl_result", :type => "radio", :value => "failure"}
+                    %input{"ng-model" => "vm.reportModel.tl_result", :type => "radio", :value => "failure"}
                     = _("Failed Events")
                   %label.radio
-                    %input{"ng-model" => "reportModel.tl_result", :type => "radio", :value => "both"}
+                    %input{"ng-model" => "vm.reportModel.tl_result", :type => "radio", :value => "both"}
                     = _("Both")
               %div{'class' => 'timeline-date-box'}
                 .input-group.bootstrap-touchspin.timeline-stepper.pull-left
                   %span.input-group-btn
-                    %button.btn.btn-default.bootstrap-touchspin-down{"ng-click" => "countDecrement()", :type => "button"} -
-                  %input.timeline-range-input.bootstrap-touchspin.form-control{"ng-model" => "reportModel.tl_range_count",
+                    %button.btn.btn-default.bootstrap-touchspin-down{"ng-click" => "vm.countDecrement()", :type => "button"} -
+                  %input.timeline-range-input.bootstrap-touchspin.form-control{"ng-model" => "vm.reportModel.tl_range_count",
                                                                                 :readonly => "readonly",
                                                                                 :type     => "text"}
                   %span.input-group-btn
-                    %button.btn.btn-default.bootstrap-touchspin-up{"ng-click" => "countIncrement()", :type => "button"} +
+                    %button.btn.btn-default.bootstrap-touchspin-up{"ng-click" => "vm.countIncrement()", :type => "button"} +
                 = select_tag("tl_range",
                   options_for_select([[_("Days"), "days"],
                                       [_("Weeks"), "weeks"],
```

Comment:
And line 79, ng-model reportModel (sorry, can't add the comment on that line for some reason)